aerospike_helpers/metrics/__init__.py

```diff
@@ -90,6 +106,58 @@ class Cluster:
     pass
 
 
+# as_node_stats has a reference to the corresponding as_node object
+# Here, we are using specific as_node fields to identify that as_node instead of storing the full as_node.
+# Since as_node has a ton of fields, we don't want to return the whole as_node.
+#
+# We also don't want to have a reference to a Node class instance
+# because our Node class has fields we don't want to expose when returning ClusterStats to the user
+# i.e Node's namespace metrics when extended metrics is disabled.
+class NodeStats:
+    """Node statistics.
+
+    Attributes:
+        name: The name of the node.
+        address: The IP address / host name of the node (not including the port number).
+        port: Port number of the node's address.
+        conns: Synchronous connection stats on this node.
+        error_count: {}
+        timeout_count: {}
+        key_busy_count: {}
+    """
+    name: str
+    address: str
+    port: int
+    conns: ConnectionStats
+    error_count: int
+    timeout_count: int
+    key_busy_count: int
+
+
+if isinstance(NodeStats.__doc__, str):
+    NodeStats.__doc__ = NodeStats.__doc__.format(
+        ERROR_COUNT_DOCSTRING,
+        TIMEOUT_COUNT_DOCSTRING,
+        KEY_BUSY_COUNT_DOCSTRING
+    )
```

Comment:
This formatting happens at run time right? Are these changes included in generated documentation?

Reply:
Yeah, the docstrings get generated at runtime:

[NodeStats](https://aerospike-python-client--820.org.readthedocs.build/en/820/aerospike_helpers.metrics.html#aerospike_helpers.metrics.NodeStats.error_count)
[NamespaceMetrics](https://aerospike-python-client--820.org.readthedocs.build/en/820/aerospike_helpers.metrics.html#aerospike_helpers.metrics.NamespaceMetrics.error_count)

I couldn't use f-strings for the docstrings in order to use those constants, so this is a workaround. (the docstring has to be a string literal)